### PR TITLE
Adds tons of new materials + cleans up existing ones

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

= Total restructuring of material folder organization and rsis.
= Adds adamantine, silver, copper, iron and hydrogen ingots with inhands and stack visualizers.
= Adds a bunch more sheet materials.
= Bunch more misc materials along with ores and such.

Draft PR for now while I put finishing touches on, just posting so people know i've camped this.

**Screenshots**

image

### CODE REVIEW

**Comment by tmtmtl30** [comment]
Do you plan on adding any functionality to these (let players make walls with them)?

**Comment by SweptWasTaken** [comment]
> 
> 
> Do you plan on adding any functionality to these (let players make walls with them)?

Nothing for now. I just want to make all these available and usable so people can start making crafting recipes with more interesting items. 

The REAL issue is coming up with a plan of how to fucking obtain these items without mining/salvage/whatever.

**Comment by Peptide90** [comment]
> Do you plan on adding any functionality to these (let players make walls with them)?

The mineral doors can easily be ported (I'm currently working on new construction recipies). But personally they're a little pointless unless players want to build a rich palace room...

Obtaining them like you say could play into asteroids, salvage and maybe into cargo thoroughfare but that's obviously down the line.

**Comment by SweptWasTaken** [comment]
Ready for review. Keep getting empty errors from StackPrototype even though all the stacks work fine so @DrSmugleaf might want to take a look at that.

**Comment by ike709** [comment]
> The REAL issue is coming up with a plan of how to fucking obtain these items without mining/salvage/whatever.

Port bluespace miners from Bee as a placeholder.

**Comment by SweptWasTaken** [comment]
> > The REAL issue is coming up with a plan of how to fucking obtain these items without mining/salvage/whatever.
> 
> 
> 
> Port bluespace miners from Bee as a placeholder.

I was actually thinking of a multi-tile thing like Void Miners from Environmental Tech.

**Comment by ike709** [comment]
Those are just multi-tile bluespace miners.

**Comment by metalgearsloth** [comment]
StorageFill stuff needs fixing from the tests.

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -22,12 +22,21 @@
...
+
 - type: Tag
   id: PlantSampleTaker
 
 - type: Tag
   id: Shovel
 
+- type: Tag
+  id: Sheet
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:39]: (Above shovel)

**Comment by SweptWasTaken** [comment]
> 
> 
> StorageFill stuff needs fixing from the tests.

fixed

**Review by metalgearsloth — APPROVED** [review]
Seems okay. If anything is broken we should add a test for it.

**Pull request merged**